blink/crossencoder/train_cross.py

```diff
@@ -60,7 +60,7 @@ def modify(context_input, candidate_input, max_seq_length):
     return torch.LongTensor(new_input)
 
 
-def evaluate(reranker, eval_dataloader, device, logger, context_length, silent=True):
+def evaluate(reranker, eval_dataloader, device, logger, context_length, zeshel=True, silent=True):
```

Comment:
Let's set the default of zeshel to be false here.

Reply:
Thanks for the review! See 85a9b3d